Cesium.CodeGen.Tests/CodeGenOperatorTests.cs

```diff
@@ -1,18 +1,18 @@
-namespace Cesium.CodeGen.Tests;
-
-public class CodeGenOperatorTests: CodeGenTestBase
-{
-    private static Task DoTest(string source)
-    {
-        var assembly = GenerateAssembly(default, source);
-
-        var moduleType = assembly.Modules.Single().GetType("<Module>");
-        return VerifyMethods(moduleType);
-    }
-
-    [Fact]
-    public Task AddIntToChar() => DoTest(@"int main() { int x = 1 + 'b'; }");
-
-    [Fact]
-    public Task SubtractIntFromInt() => DoTest(@"int main() { int x = 2 - 1; }");
-}
+namespace Cesium.CodeGen.Tests;
+
+public class CodeGenOperatorTests: CodeGenTestBase
+{
+    private static Task DoTest(string source)
+    {
+        var assembly = GenerateAssembly(default, source);
+
+        var moduleType = assembly.Modules.Single().GetType("<Module>");
+        return VerifyMethods(moduleType);
+    }
+
+    [Fact(Skip = "Implicit casts should be part of IR in order to restore this test.")]

```

Comment:
A bit scary touch this, since either I have to provide bandaid of unknown complexity, or push towards #201 which may be complex. Right now this is mostly renaming, and even here I manage to add some refactoring. Too complex to my taste.

Reply:
Alright, let's fix it separately, then.